### ISSUE TITLE
False positive `expect-expect` and `valid-title` on `test.extend`

### ISSUE THREAD

**Issue by akellbl4** [vitest-dev/eslint-plugin-vitest#495]
When the context of `test` or `it` is extended linter considers that code as a test case with an invalid title and absent `expect`

Any combination with different names for tests is not working, even if the extended `it/test` is called with a different name other than the original.

**Example**
```ts
import { it as base } from 'vitest'

// eslint-disable-next-line vitest/expect-expect
const it = base.extend<{
  foo: string,
  bar: string
  // eslint-disable-next-line vitest/valid-title
} >({
 async foo({}, use) {
   await use('foo')
 },
 async bar({}, use) {
   await use('bar')
 }
})
```

**Env**
eslint: 8.56.0
eslint-plugin-vitest: 0.5.4

**Configuration**
```js
module.exports = {
  overrides: [
    {
      files: ['*.test.{ts,tsx}'],
      plugins: ['vitest'],
      extends: ['plugin:vitest/legacy-recommended']
    }
  ],
}
```

**Comment by akellbl4** [comment]
I see that it should be handled [here](https://github.com/veritem/eslint-plugin-vitest/blob/main/src/rules/expect-expect.ts#L93), not sure why it doesn't work :/

**Comment by veritem** [comment]
I don't know why this is a problem for you. I can't reproduce it. I also went a head and added you exact example as a test case. https://github.com/vitest-dev/eslint-plugin-vitest/pull/530/files#diff-abe2f05f8778f2a2e6dd85401e32ad8e399b2f13d9701a108108d7db3ad8a26cR26-R39
